src/lib/core/managers/download.svelte.ts

```
@@ -0,0 +1,249 @@
+  private parseDownloadOutput(output: string) {
+    const MiBtoBytes = (mib: number) => mib * 1024 * 1024;
+    const timeToMs = (timeStr: string) => {
+      const [h, m, s] = timeStr.split(':').map(Number);
+      return ((h * 3600) + (m * 60) + s) * 1000;
+    };
+
+    const result: Partial<DownloadProgress> = {};
+
+    let match = output.match(/Install size: ([\d.]+) MiB/);
+    if (match) {
+      result.installSize = MiBtoBytes(parseFloat(match[1]));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)